tests/requirements-win.txt

```diff
@@ -1,3 +1,5 @@
+--extra-index-url https://pypi.anaconda.org/giumas/simple
+--extra-index-url https://pypi.anaconda.org/xoviat/simple
```

Comment:
Why do we need this here? All pacakges in this fail ought to be available at PyPI, i Thought?